implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -430,16 +444,26 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
         contextInfo.setMaxTimeToLiveSecs(maxTimeToLiveSecs.orElse(null));
         contextInfo.setJwksRefreshInterval(jwksRefreshInterval.orElse(null));
         contextInfo.setForcedJwksRefreshInterval(forcedJwksRefreshInterval);
-        if (signatureAlgorithm.orElse(null) == SignatureAlgorithm.HS256) {
-            throw ConfigMessages.msg.hs256NotSupported();
+        if (mpJwtVerifierPublicKeyAlgorithm.isPresent()) {
```

Comment:
I propose to optimize this block a bit, have an in scope property set to either mp jwt or smallrye prop, and do HS256 check once

Reply:
Ok